stylesheets/_emoji.scss

```diff
@@ -111,14 +111,15 @@ button.emoji {
 
 .emoji-panel-container {
   height: 0px;
-  .ep-container {
-    padding-bottom: 5px;
-  }
```

Comment:
I think you want at least a pixel between the bottom of the scroll handle and the blue outline on the message field.

Reply:
I prefer no buffer; I think it makes things cleaner. But we'll see what the team says when they get a chance to play with it!